rgbd_image_buffer/src/image_buffer.cpp

```diff
@@ -124,9 +126,9 @@ bool ImageBuffer::waitForRecentImage(rgbd::ImageConstPtr& image, geo::Pose3D& se
     if (timeout_tries <= 0)
     {
         ROS_DEBUG_STREAM_NAMED("iamge_buffer", "[IMAGE_BUFFER](waitForRecentImage) defaulting to 10 tries instead of '" << timeout_tries << "'");
```

Comment:
```suggestion
        ROS_DEBUG_STREAM_NAMED("iamge_buffer", "[IMAGE_BUFFER](waitForRecentImage) defaulting to 25 tries instead of '" << timeout_tries << "'");
```